src/main/resources/application.properties

```diff
@@ -11,6 +11,7 @@ spring.datasource.password=pass
 
```

Comment:
1. I don't see a reason to customize Hikari right now.
2. `spring.kafka.bootstrap-servers` can be removed
3. remove obsolete configuration properties like MySQL config, dialect, etc.
4. only tests should know about H2 (use application-test.yml)
5. it's reasonable to use environment variables for `datasource` config
6. `spring.jpa.hibernate.ddl-auto`  -> set to `validate` to ensure Liquibase schema matches with Hibernate